README.md

```diff
@@ -107,6 +108,21 @@ quote.change_percent_s # '+0.42%'
 
 See [#quote](https://iexcloud.io/docs/api/#quote) for detailed documentation or [quote.rb](lib/iex/resources/quote.rb) for returned fields.
 
+### Stream quotes
+
+Streams quotes in real-time.
+
+`interval` option lets you limit amount of updates. Possible values: `1Second 5Second 1Minute`
```

Comment:
The `interval` ... so it's a complete sentence.

Possible values are ... and quote each value. End with a period.